FRAuth/FRAuth/WebAuthn/FRWebAuthn.swift

```diff
@@ -0,0 +1,30 @@
+// 
+//  FRWebAuthn.swift
+//  FRAuth
+//
+//  Copyright (c) 2023 ForgeRock. All rights reserved.
+//
+//  This software may be modified and distributed under the terms
+//  of the MIT license. See the LICENSE file for details.
+//
+
+
+import Foundation
+import FRCore
+
+public class FRWebAuthn: NSObject {
```

Comment:
Missing doc?

Reply:
yeah we need a doc here , since its publicly exposed